src/docs/issue-platform.mdx

```diff
@@ -53,67 +63,92 @@ class ProfileFileIOGroupType(GroupType):
 ```
 
 ### Register a Category
+
 You might also want to register a new category related to your issue type as well. This allows you to group related issue types together. To do so, add your category to <Link to="https://github.com/getsentry/sentry/blob/97f4174c28041357aa190f27012dc33c1c7bafc7/src/sentry/issues/grouptype.py#L17-L22">this enum</Link>
 
-## Send Occurrence or Status Change Payloads to the Issue Platform
+## Send Payloads to the Issue Platform
+
 If operating within the Sentry codebase, you can call <Link to="https://github.com/getsentry/sentry/blob/master/src/sentry/issues/producer.py#L41-L46">sentry.issues.produce_occurrence_to_kafka</Link> with the following optional parameters:
+
 - `payload_type`: <Link to="#payload-type">PayloadType</Link>
-- `occurrence`: <Link to="https://github.com/getsentry/sentry/blob/d262bbd9d8d3a2747d5251bee35c016c13ca050a/src/sentry/issues/issue_occurrence.py#L58">IssueOccurrence</Link>
-- `status_change`: <Link to="#status-change-message">StatusChangeMessage</Link>
+- one of the following, per `payload_type`
+  - `occurrence`: <Link to="https://github.com/getsentry/sentry/blob/d262bbd9d8d3a2747d5251bee35c016c13ca050a/src/sentry/issues/issue_occurrence.py#L58">IssueOccurrence</Link>
+  - `status_change`: <Link to="https://github.com/getsentry/sentry/blob/7fd40bc33607e727ea2fac0183bd04f4ad298ba3/src/sentry/issues/status_change_message.py#L15">StatusChange</Link>
 - `event_data`: Dict[str, Any]
 
-
 If operating from an external service then pass a json payload that matches the <Link to="#issue-occurrence-schema">Issue Occurrence Schema</Link> to the `ingest-occurrences` topic on the <Link to="https://github.com/getsentry/getsentry/blob/97039335a29517a7e48149148cdd7a30a60971e7/getsentry/conf/settings/prod.py#L567-L571">issue occurrences Kafka cluster</Link>
 
+### Payload Type
```

Comment:
nit: feel like the h3 here should have a colon like the others (ie. issue occurrence schema) or they should all not have colons